### PR TITLE
fibonacci working

### PR DESCRIPTION
**Pull request by shadston**

### CODE REVIEW

File: module1/homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -1,11 +1,32 @@
...
 
 int fibonacci_recursive(int sequence) {
-    // TODO: Your implementation goes here
-    return 0;
+    if(sequence == 0) {
+        return 0;
+    }
+    if(sequence == 1) {
+        return 1;
+    }
```
Comment (Kasperice) [inline module1/homework/fibonacci/fibonacci.hpp:30]: You can notice some dependence between `sequence` and return value. Then you can simplify your solution

Comment (shadston) [reply module1/homework/fibonacci/fibonacci.hpp:29]: ok thank you

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
```
Comment (nauka-programowania-MA) [inline module1/homework/fibonacci/fibonacci.hpp:27]: space after if

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1430], "points": [2, 6] }

**Pull request closed without merging**